### DISCUSSION TITLE
Need guidance in setting dynamic attachment name for multiple attachment

### DISCUSSION THREAD

**Discussion by mohanbala23599** [emailjs-com/help#113] [Q&A]
Using dynamic attachment and attachment type variable attachment. We are attaching multiple attachment to the email. But we are not able to set the dynamic file name for each attachment attached. We are stuck in this issue for past 10 days. Need your guidance in fixing this issue.

We are sending attachment as array
attachment - ['pdf-url','pdf-url']
But attachment name is not accepting in this array.

Please respond asap. Thanks in advance.

Language using - React JS
EmailJS - Business version

**Comment by xr0master** [comment]
Hi,
Unfortunately, there is no such option for attachments in an array yet.

You can create multiple dynamic variables for attachments and use them instead of an array.
In this case, you will have control over the filenames for each attachment.
